src/ansys/fluent/core/codegen/datamodelgen.py

```diff
@@ -317,11 +332,11 @@ def _write_static_info(self, name: str, info: Any, f: FileIO, level: int = 0):
                 pass
         for k in parameters:
             k_type = _PY_TYPE_BY_DM_TYPE[info["parameters"][k]["type"]]
```

Comment:
Please use the datamodel type in comparison:
```
k_type = info["parameters"][k]["type"]
if k_type in ["String", "String List", "ListString"]:
    etc.
```

Reply:
Yes, we need to cover all types in "_PY_TYPE_BY_DM_TYPE" here ideally.

Reply:
@mkundu1 Done. Thank you.